lib/cocoapods/installer/project_cache/target_cache_key.rb

```diff
@@ -136,7 +136,7 @@ def self.from_pod_target(sandbox, pod_target, is_local_pod: false, checkout_opti
             'PROJECT_NAME' => pod_target.project_name,
           }
           if is_local_pod
-            relative_file_paths = pod_target.all_files.map { |f| Pathname.new(f).relative_path_from(sandbox.root).to_s }
+            relative_file_paths = pod_target.all_files.map { |f| f.relative_path_from(sandbox.root).to_s }
```

Comment:
`pod_target.all_files` now returns `Array<Pathname>` which is the more common use case